### ISSUE TITLE
Untracked keys

### ISSUE THREAD

**Issue by nbw** [nbw/opz#1]
When pressing the stop button.

```
[OPZ]: 176,123,0
[OPZ]: 177,123,0
[OPZ]: 178,123,0
[OPZ]: 179,123,0
[OPZ]: 180,123,0
[OPZ]: 181,123,0
[OPZ]: 182,123,0
[OPZ]: 183,123,0
[OPZ]: 184,123,0
[OPZ]: 185,123,0
[OPZ]: 186,123,0
[OPZ]: 187,123,0
[OPZ]: 188,123,0
[OPZ]: 189,123,0
[OPZ]: 190,123,0
[OPZ]: 191,123,0
```

**Comment by nbw** [comment]
Related to dials. 

Errors on value(input) related to the "kill" dials midi note.

**Comment by nbw** [comment]
Resolved #2
